packages/bsky/src/hydration/hydrator.ts

```diff
@@ -442,6 +470,9 @@ export class Hydrator {
       ctx.viewer
         ? this.feed.getPostViewerStates(threadRefs, ctx.viewer)
         : undefined,
+      ctx.includeThreadRootAuthorState
+        ? this.feed.getThreadRootAuthorStates(threadRefs)
```

Comment:
I wonder if it would be more straightforward leave `getThreadRootAuthorStates()` out of generic post hydration.  It is only needed on posts within threads, so perhaps it could be added within `hydrateThreadPosts()`.